metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -1275,7 +1311,13 @@ def _create_lineage_aspect_for_dbt_node(
                     node.name,
                     self.config.target_platform,
                     self.config.env,
-                    self.config.platform_instance,
+                    match_target_platform_instance(
+                        self.config.target_platform_instance,
+                        None,
```

Comment:
why set this as none here? let's let the logic of the method handle whether to use the dbt platform instance or not. that will make things less error prone

Reply:
Ok I added reference to config platform_instance here. Originally here there was a None if I recall correctly and I wanted to preserve the original logic, so that in case platform would match `dbt` it would still return None as before. But if you think we should return platform instance I adjusted it to return platform instance.